.gitignore

```diff
@@ -11,3 +12,10 @@ logInsatall.log
 symfony/test/functional/node_modules
 symfony/tests/functional/yarn.lock
 coverage/
+.vscode
```

Comment:
remove this line since rule already added 2nd line

Reply:
Removed .vscode in line 15